### PR TITLE
Emml2584 corr mk

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
I haven't seen the ms. so I am using the catalog description.
[EMML 2584.pdf](https://github.com/BetaMasaheft/Manuscripts/files/7730503/EMML.2584.pdf)

### CODE REVIEW

File: EMML/2001-3000/EMML2584.xml [hunk 1]
```diff
@@ -32,11 +32,102 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
             <msDesc xml:id="ms">
                <msIdentifier>
                   <repository ref="INS0004HMML"/>
-                  <collection>Parish Church, Ankober</collection>
```
Comment (DariaElagina) [inline EMML/2001-3000/EMML2584.xml:35]: Thank you, Marcin. I think that it has been decided to substitute the HMML rep with the real repository of the manuscripts and keep information on the  HMML in the collection. Here in the example 2: https://betamasaheft.eu/Guidelines/?q=EMML&id=identifiersStructure

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: EMML/2001-3000/EMML2584.xml (hunk 1 above)
Comment (MarcinKrawczuk) [reply EMML/2001-3000/EMML2584.xml:35]: Done!

File: EMML/2001-3000/EMML2584.xml [hunk 2]
```diff
@@ -32,18 +32,18 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
...
+                  <collection></collection>
```
Comment (eu-genia) [inline EMML/2001-3000/EMML2584.xml]: the repository and collection should be, as in the example 
```
<msIdentifier>
                        <repository ref="********"> (PUT HERE ID FOR THE ANKOBER CHURCH IF KNOWN)</repository>
                        <collection>Ethiopian Manuscript Microfilm Library</collection>
                        <idno>EMML no. 2584</idno>
                    </msIdentifier>
```

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
I think @DariaElagina must approve for this request to be merged, as she commented previously

File: EMML/2001-3000/EMML2584.xml [hunk 3]
```diff
@@ -31,7 +31,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
          <sourceDesc>
             <msDesc xml:id="ms">
                <msIdentifier>
-                  <repository ref="INS0619AMX"/>
+                  <repository ref="INS0004HMML"/>
```
Comment (eu-genia) [inline EMML/2001-3000/EMML2584.xml:34]: no it was all ok, I had deleted my comment, I had overseen that you already fixed it. Ankobar is correct as repository

Comment (eu-genia) [reply EMML/2001-3000/EMML2584.xml:34]: please change back to INS0619AMX ) I had approved but I cannot merge without Daria

**Review by eu-genia — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**